styles.css

```diff
@@ -0,0 +1,138 @@
+.container {
+	background-image: url("https://backgrounddownload.com/wp-content/uploads/2018/09/background-banner-motor.jpg");
```

Comment:
save a local copy into your project and link to that as its safer than relying on the other website to always be there